tests/smoke/support/cta/files/misc.ts

```diff
@@ -14,15 +14,34 @@ export const navigateToFolder = async ({
   }
 }
 
+/**
```

Comment:
to have a timeout here is kinda okish, but we should keep track of it and replace it with something that checks that the network requests are through